pod/video/utils.py

```diff
@@ -56,8 +56,12 @@
 SECURE_SSL_REDIRECT = getattr(settings, "SECURE_SSL_REDIRECT", False)
 
 
+# ##########################################################################
```

Comment:
Merci de laisser les pydocs au bon format ("""Describe the func.""") :)

Reply:
Mauvaise manip de ma part, je corrige